src/hardware/pwm.rs

```diff
@@ -78,6 +78,7 @@ impl Pwm {
     const MAX_DUTY: u32 = 10000; // Maximum duty cycle valid for all channels.
     const V_PWM: f32 = 3.3; // MCU PWM pin output high voltage
     pub const MAX_CURRENT_LIMIT: f32 = (Pwm::V_PWM * 0.15) / (VREF_TEC * R_SENSE);
```

Comment:
I would put the Datasheet maximum value of 3 A here. I think that's more relevant that the equivalent of 100% duty cycle.

Reply:
Good